mlir/test/rocmlir-gen/generate-clone-harness.mlir

```diff
@@ -0,0 +1,19 @@
+// RUN: rocmlir-gen -fut mlir_reshape_convolution  --arch %arch --clone-harness %s| rocmlir-driver -kernel-pipeline=migraphx -host-pipeline=migraphx,highlevel |rocmlir-gen -ph -rand 1 -rand_type float -fut mlir_reshape_convolution_wrapper --verifier clone - | FileCheck %s
+
```

Comment:
Can you add another // RUN line with the kernel lowering pipeline to check whether an actual kernel is produced (a.k.a. attached to a function as a blob) ? 

so its not a cpu vs cpu verfication. This is because we had several instances of the said false verification.